.github/templates/main.yml

```diff
@@ -290,7 +290,20 @@ jobs:
       - name: Set version suffix
         id: set-version-suffix
         #! Build suffix is PR-1234.5 for PR builds or alpha.123 for branch builds.
-        run: echo "::set-output name=build_suffix::${{ github.event_name == 'pull_request' && format('PR-{0}', github.event.number) || 'alpha'}}.${{ github.run_number }}"
+        run: |
+          $suffix = ""
+          if ($env:GITHUB_EVENT_NAME -eq "pull_request")
+          {
+            if (-Not $env:GITHUB_REF.Contains("release"))
+            {
+              $suffix = "PR-${{ github.event.number }}.$env:GITHUB_RUN_NUMBER"
```

Comment:
I'd have like to use the env syntax, since it's easier to read to my eyes, but `$env:GITHUB_EVENT_NUMBER` doesn't seem to produce any output, so I went back to `${{ github.event.number }}`, just because it's the only way to read that value.